src/app/parcels/parcelsTable/ParcelsTable.tsx

```diff
@@ -226,15 +225,22 @@ const ParcelsTable: React.FC<ParcelsTableProps> = ({
         }
     }, [areFiltersLoadingForFirstTime, fetchAndDisplayParcelsData]);
 
-    const packingManagerViewDataPortion = (
-        parcelsDataPortion: ParcelsTableRow[]
-    ): ParcelsTableRow[] => {
-        return parcelsDataPortion.filter((parcel) => {
-            if (shouldBeInPackingManagerView(parcel, today, yesterday)) {
-                return parcel;
-            }
-        });
-    };
+    const packingManagerViewDataPortion = React.useCallback(
+        (parcelsDataPortion: ParcelsTableRow[]): ParcelsTableRow[] => {
+            return parcelsDataPortion.filter((parcel) => {
+                if (shouldBeInPackingManagerView(parcel, today, yesterday)) {
+                    return parcel;
+                }
+            });
+        },
+        [today, yesterday]
+    );
+
+    useEffect(() => {
+        isPackingManagerView
+            ? setFilteredParcelCount(packingManagerViewDataPortion(parcelsDataPortion).length)
+            : setFilteredParcelCount(parcelsDataPortion.length);
+    }, [isPackingManagerView, parcelsDataPortion, packingManagerViewDataPortion]);
```

Comment:
Is there a way we can do this without an effect? This use case is pretty much one of the examples of when you shouldn't use an effect in the [React docs](https://react.dev/learn/you-might-not-need-an-effect#updating-state-based-on-props-or-state)